CONTRIBUTION.md

```diff
@@ -73,28 +73,35 @@ Please follow the instructions of how to [Building Slang from Source](docs/build
 For a quick reference, follow the instructions below.
 
 #### Windows
-Find where the premake executable is. The location may change when we upgrade Premake.
-```
-C:\git\slang> where /r external\slang-binaries\premake premake*.exe
-```
+Download and install CMake from [CMake.org/download](https://cmake.org/download)
 
-Run the premake with the following command line options after replacing "PREMAKE" with the result from the previous command.
+Run CMake with a following command to generate Makefile:
 ```
-# For VisualStudio 2019
-C:\git\slang> PREMAKE vs2019 --deps=true --arch=x64
-# For VisualStudio 2017
-C:\git\slang> PREMAKE vs2017 --deps=true --arch=x64
+# For VisualStudio 2022
+C:\git\slang> cmake.exe --preset vs2022
 ```
 
 Open slang.sln with VisualStudio IDE and build it for "x64".
 
+Or you can build with a following command:
+```
+C:\git\slang> cmake.exe --build --preset release
+```
+
 #### Linux
-Find where the premake is and run it with gmake2 option.
+Install CMake and Ninja.
+```
+$ sudo apt-get install cmake ninja-build
```

Comment:
I wonder if it would be better if we would suggest to install cmake over `pip install`. `pip` always has the latest version of CMake.

The problem with `apt` is that most repos have a old versions of CMake. For example, Ubuntu 22 and older do not have cmake 3.25 in their default apt repos list.

This would probably stop the whole "why is cmake not working" issue.